### ISSUE TITLE
Imprisonment Spell Format Issue

### ISSUE THREAD

**Issue by FictionRaider007** [Orcpub/orcpub#454]
**Describe the bug**

The "Imprisonment" spell on the DMV website has a mistake in it. In the "Duration" part of the spell's description the first paragraph or so of the spell follows it when it should come after "Components"

**To Reproduce**

1. Go to 'Spell List'
2. Type in 'Imprisonment'
3. Click on 'Imprisonment'

**Expected behavior**

To see the spell description displayed normally.

**Additional context**

I'm not really sure if this counts as a "bug" but the last time I found a typo I posted it on reddit, after which they told me to come here to write out any other errors I encountered. Hope I did the right thing.

**Comment by DivertedCircle** [comment]
Screenshots help. To clarify, Duration should simply read 'Until dispelled'. Everything in the duration section that currently follows needs to be moved to description.

TLDR: Red box needs to be moved to description.

![image](https://user-images.githubusercontent.com/26470570/93841019-8b6cca00-fc4f-11ea-871c-8c24f5987b98.png)
